src/bindings/python/src/compatibility/openvino/inference_engine/ie_api.pyx

```diff
@@ -1113,7 +1113,7 @@ cdef class ExecutableNetwork:
         """
         A tuple of :class:`InferRequest` instances
         """
-        cdef size_t c_infer_requests_size = deref(self.impl).infer_requests.size()
+        cdef int c_infer_requests_size = deref(self.impl).infer_requests.size()
```

Comment:
Why did you change the returning type from size_t to int?

I think `deref(self.impl).infer_requests.size()` should return `size_t`, am I wrong?

Reply:
Discussed offline

Reply:
I have warning in autogenerated code otherwise